### DISCUSSION TITLE
iOS Tutorial `urlRequest.setValue()` incorrect use in docs

### DISCUSSION THREAD

**Discussion by Richenglish** [deepgram/community#143] [General help] [answered]
Hi,

I'm attempting to build a Swift app and are starting with the iOS tutorial on the Deepgram blog. 
https://blog.deepgram.com/ios-live-transcription/

Building and running the example application generates the error: 
notAnUpgrade(401)
WSError(type: Starscream.ErrorType.protocolError, message: "masked and rsv data is not currently supported", code: 1002)

Any ideas?

**Comment by Richenglish** [comment, marked as answer]
I've solved the issue I was having.

In the tutorial, the line
`urlRequest.setValue(apiKey, forHTTPHeaderField: "Authorization")`

should be 
`urlRequest.setValue("Token \(apiKey)", forHTTPHeaderField: "Authorization")`

**Comment by lukeocodes** [comment]
Thanks Rich, we really appreciate you finding and responding with the solution. We've made a point to update our docs!
